src/main/java/de/dataelementhub/model/service/ExportService.java

```diff
@@ -78,4 +92,15 @@ public List<ExportInfo> allExports(int userId, String exportDirectory) {
     }
     return exportDescriptions;
   }
+
+  /**
+   * Get export directory.
+   */
+  public String getExportDirectory() {
+    if (defaultExportDirectory == null) {
```

Comment:
Just a question concerning the nomenclature. Wouldn't a **default** directory be the fallback and never be null? So shouldn't the "defaultExportDirectory" be called the "exportDirectory" and the fallback (java.io.tmpdir) be the default directory?

Reply:
I agree with you. The prefix should be deleted because the default directory must not be null